core/healthcheck/interfaces/controllers/healthcheck.go

```diff
@@ -29,19 +35,46 @@ type (
 	}
 )
 
+const (
+	statusMeasureChecksName   = "flamingo/status/checks"
+	statusMeasureFailuresName = "flamingo/status/failures"
```

Comment:
should we use something ending in total? see https://prometheus.io/docs/practices/naming/#metric-and-label-naming

Reply:
done